editor/src/clj/editor/code/view.clj

```diff
@@ -1773,6 +1773,64 @@
                                                   (get-property view-node :cursor-ranges)
                                                   (get-property view-node :regions)))))
 
+(handler/defhandler :goto-definition :code-view
+  (enabled? [view-node]
```

Comment:
I think we may be able to add an `evaluation-context` argument here for us to use inside the `enabled?` clause?

Reply:
Oh, good catch!